packages/twenty-server/src/engine/metadata-modules/remote-server/utils/build-update-remote-server-raw-query.ts

```diff
@@ -0,0 +1,120 @@
+import { isDefined } from 'class-validator';
+
+import {
+  RemoteServerEntity,
+  RemoteServerType,
+} from 'src/engine/metadata-modules/remote-server/remote-server.entity';
+
+export type DeepPartial<T> = {
+  [P in keyof T]?: DeepPartial<T[P]>;
+};
+
+export const updateRemoteServerRawQuery = (
```

Comment:
suffix file name with .utils

Reply:
and please separate the functions that create the userMapping and the other options